pino.js

```diff
@@ -44,6 +57,10 @@ function pino (opts, stream) {
     buf: ''
   }
 
+  if (opts.enable === false) {
```

Comment:
....maybe.. opts.silent ? so then `if (opts.silent)  { }`

also - should there be `enable` and `disable` methods, so logging can be turned back on